example/driver/simpledriver.go

```diff
@@ -26,7 +26,7 @@ type SimpleDriver struct {
 
 // DisconnectDevice handles protocol-specific cleanup when a device
 // is removed.
-func (s *SimpleDriver) DisconnectDevice(deviceName string, protocols map[string]map[string]string) error {
+func (s *SimpleDriver) DisconnectDevice(deviceName string, protocols map[string]ds_models.ProtocolProperties) error {
```

Comment:
Why use ds_models here? This file doesn't import models from core-contracts so there's no conflict. Also use of "_"s names is typically not done in Go.

Reply:
it was imported core-contracts, so I forgot to remove the alias.
ok, I will open another issue to replace all ds_model with dsModel in SDK